pkg/crossnamespace/validation.go

```diff
@@ -0,0 +1,120 @@
+package crossnamespace
```

Comment:
I think you will need to add the header for the license here to fix the linter warnings